WordPress/src/main/java/org/wordpress/android/ui/publicize/PublicizeWebViewFragment.java

```diff
@@ -118,6 +121,17 @@ public void onResume() {
         setNavigationIcon(R.drawable.ic_close_white_24dp);
     }
 
+    private void clearCookies() {
```

Comment:
Would it be better to clear cookies from specific domains rather than clear them all? If so, [this](http://stackoverflow.com/a/7802460/1673548) may help.